falco/templates/_helpers.tpl

```diff
@@ -229,3 +229,70 @@ be temporary and will stay here until we move this logic to the falcoctl tool.
     - mountPath: /gvisor-config
       name: falco-gvisor-config
 {{- end -}}
+
+
+{{- define "falcoctl.initContainer" -}}
+- name: falcoctl-artifact-install
+  image: {{ printf "%s/%s:%s" .Values.falcoctl.image.registry .Values.falcoctl.image.repository .Values.falcoctl.image.tag }}
```

Comment:
I think it's better to create a helper here. We already do that for the other images. For example :point_down: 

https://github.com/falcosecurity/charts/blob/9b27a5925f4c8a4d0fd500a2e963d572e2cce767/falco/templates/pod-template.tpl#L55